packages/@lwc/engine-core/src/framework/def.ts

```diff
@@ -96,11 +91,9 @@ function getCtorProto(Ctor: any, subclassComponentName: string): ComponentConstr
 
 function createComponentDef(
     Ctor: ComponentConstructor,
-    meta: ComponentMeta,
     subclassComponentName: string
```

Comment:
what is the purpose of subclassComponentName at this point?

Reply:
Looks like we were just passing around so that it could be used in some error logs inside `getCtorProto()`. I went ahead and removed all that!